src/utils/variant.ts

```diff
@@ -1,19 +1,6 @@
-export const types: string[] = [
-    'betty',
-    'blockfish',
-    'brinely',
-    'clayfish',
-    'dasher',
-    'flopper',
-    'glitter',
-    'kob',
-    'snooper',
-    'spotty',
-    'stripey',
-    'sunstreak',
-];
+export const types: string[] = ['betty', 'blockfish', 'brinely', 'clayfish', 'dasher', 'flopper', 'glitter', 'kob', 'snooper', 'spotty', 'stripey', 'sunstreak'];
 export const typesMapping: Record<string, [number, number]> = {
-    // first least byte / second-least byte : https://minecraft.fandom.com/wiki/Tropical_Fish (Entity data)
+    // first least byte / second-least byte : https://minecraft.fandom.com/wiki/Tropical_Fish#Entity_data
```

Comment:
Probably best to point to the independent wiki [wiki](https://minecraft.wiki/w/Tropical_Fish#Entity_data), the fandom one is abandoned and unmaintained

Reply:
Yes thanks, I updated it